javascripts/discourse/components/kanban/card.gjs

```diff
@@ -0,0 +1,184 @@
+import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
+import { getOwner } from "@ember/application";
+import { hash } from "@ember/helper";
+import { on } from "@ember/modifier";
+import { action } from "@ember/object";
+import { inject as service } from "@ember/service";
+import { htmlSafe } from "@ember/template";
+import PluginOutlet from "discourse/components/plugin-outlet";
+import TopicStatus from "discourse/components/topic-status";
+import concatClass from "discourse/helpers/concat-class";
+import { renderAvatar } from "discourse/helpers/user-avatar";
+import renderTag from "discourse/lib/render-tag";
+import icon from "discourse-common/helpers/d-icon";
+
+export default class KanbanCard extends Component {
+  @service kanbanManager;
+
+  @tracked dragging;
+
+  // TODO - FIX THIS ONCE CORE EXPORTS IT PROPERLY
+  formatDate = getOwner(this).resolveRegistration("helper:format-date");
+  categoryBadge = getOwner(this).resolveRegistration("helper:category-badge");
```

Comment:
Core exports these properly now, so we should be able to import as normal helpers in gjs (and update the template to remove `this.` from these two)